CrazyCanvas/Source/States/ServerState.cpp

```diff
@@ -65,6 +66,8 @@ bool ServerState::OnServerDiscoveryPreTransmit(const LambdaEngine::ServerDiscove
 	ServerBase* pServer = event.pServer;
 
 	pEncoder->WriteUInt8(pServer->GetClientCount());
+	pEncoder->WriteString(m_ServerName);
+	pEncoder->WriteString("Bajskorv");
```

Comment:
Bajskorvs in master. Rude!

Reply:
Already fixed ^^ But admit, it made you smile a little